cmd/gokey/main.go

```diff
@@ -79,20 +83,43 @@ func genKey(seed []byte, w io.Writer) {
 	}
 }
 
-// TODO: parametrize size
-// generates raw 32 bytes
 func genRaw(seed []byte, w io.Writer) {
+	if isInvalidLength(length) {
+		log.Fatalln("Invalid Length")
+	}
+	if !isFlagPassed("l") {
+		length = 32
+	}
+
 	raw, err := gokey.GetRaw(pass, realm, seed, unsafe)
 	if err != nil {
 		log.Fatalln(err)
 	}
 
-	_, err = io.CopyN(w, raw, 32)
+	_, err = io.CopyN(w, raw, int64(length))
 	if err != nil {
 		log.Fatalln(err)
 	}
 }
 
+func isFlagPassed(name string) bool {
```

Comment:
the `flag` package has as `Set()` function to set a flag, so let's rename this function to `isFlagSet()` to be consistent

Reply:
Resolved in a4ea30a